### ISSUE TITLE
[AIR 50.2.4.2][Android] `StageText` broken (not showed on second time and `StageText::text` returns null)

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#3013]
## Problem Description
`StageText` broken using AIR 50.2.4.2 for Android.
Simple case with show-hide-show for `StageText` makes it not shown anymore. Moreover `StageText::text` returns `null`.
It makes `StageText` completely unusable.

Tested with latest AIR 50.2.4.2 with different AIR applications, different devices, architectures and Android versions.
Testew with and without `<runtimeInBackgroundThread>true</runtimeInBackgroundThread>`.
Same issues in all cases.
There is no such issues with Windows.
Not tested with iOS, macOS and Linux.

May be related issue: https://github.com/airsdk/Adobe-Runtime-Support/issues/2888

## Steps to Reproduce
Launch application with code below with Android device. It just show `StageText`, then remove it from stage and set stage again (to show it again with new text).

Application example with sources attached.
[android_stagetext_bugs.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/13978419/android_stagetext_bugs.zip)

```
package {
	import flash.display.Sprite;
	import flash.events.Event;
	import flash.geom.Rectangle;
	import flash.text.StageText;
	
	public class AndroidStageTextBugs extends Sprite {
		private var stageText:StageText = new StageText();
		
		public function AndroidStageTextBugs() {
			addEventListener(Event.ADDED_TO_STAGE, addedToStage);
		}
	
		private function addedToStage(e:Event):void {
			removeEventListener(Event.ADDED_TO_STAGE, addedToStage);
			
			stageText.fontSize = 40;
			stageText.viewPort = new Rectangle(100, 100, 200, 200);
			stageText.text = "text";
			stageText.stage = stage;
			trace("text:", stageText.text);//text
			stageText.stage = null;
			stageText.stage = stage;
			trace("text:", stageText.text);//null
			//StageText not shown anymore, even if try to set up text again:
			stageText.text = "text2";
		}
	}
}
```

**Actual Result:**
`StageText` not shown anymore. `StageText::text` returns `null`.
In traces you will see:
```
text: text
text: null
```

**Expected Result:**
`StageText` shown. `StageText::text` returns current text.
In traces you will see:
```
text: text
text: text
```

## Known Workarounds
Downgrade to AIR SDK 50.2.4.1.

**Comment by itlancer** [comment]
@ajwfrost 
Issue still exists with latest AIR 50.2.4.3.

**Comment by ajwfrost** [comment]
I think we have a fix proposal for this one - similar issue to #3000 
thanks

**Comment by itlancer** [comment]
Fixed with latest AIR 50.2.4.4.
Thanks!
